### PR TITLE
Test choice screen 

### PR DESCRIPTION
**Pull request by Ak216puniA**

### CODE REVIEW

File: src/components/confirmationDialog/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                    onClick={confirmAction}
+                    className="action-button confirm-action"
+                    >
+                        {confirmButtonText}
+                    </Button>
+                </DialogActions>
+            </Dialog>
+        </div>
+    )
+}
```
Comment (nikkuAg) [inline src/components/confirmationDialog/index.js:58]: End mai line add krde

File: src/components/header/index.scss [hunk 2]
```diff
@@ -15,6 +15,8 @@
 			color: #ffffff;
 			margin-right: 1rem;
 			font-family: "Poppins", sans-serif;
+			// added
```
Comment (nikkuAg) [inline src/components/header/index.scss:18]: comment hta de

File: src/constants/tableHeader.js [hunk 3]
```diff
@@ -125,14 +126,19 @@ export const choicesHeader = [
...
 	{
 		label: "Quota",
 	},
+	// renamed
+	// {
+	// 	label: "Seat Pool",
+	// },
 	{
-		label: "Seat Pool",
+		label: "Gender",
```
Comment (nikkuAg) [inline src/constants/tableHeader.js:141]: Why changed to Gender

File: src/constants/tableHeader.js
Comment (nikkuAg) [inline src/constants/tableHeader.js:1]: Remove all comments

File: src/containers/TestChoice/index.js
Comment (nikkuAg) [inline src/containers/TestChoice/index.js:1]: Can we divide this component in more than one component. Ek file mai code jada ho rha ha

File: src/containers/list.scss
Comment (nikkuAg) [inline src/containers/list.scss:1]: Remove comments

File: src/index.js
Comment (nikkuAg) [inline src/index.js:1]: Commment hta de

**Review by nikkuAg — CHANGES_REQUESTED** [review]
Har file mai last mai ke new line add krde

File: src/constants/tableHeader.js (hunk 3 above)
Comment (Ak216puniA) [reply src/constants/tableHeader.js:141]: Cause design me change kia hua hai....isley yahan change kia hai

**Pull request merged**